src/patient-registration/input/custom-input/openmrs-autosuggest/openmrs-autosuggest.component.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useRef, useState } from 'react';
+import { Search } from 'carbon-components-react';
+import styles from './openmrs-autosuggest.css';
+import { openmrsFetch } from '@openmrs/esm-api';
+
+interface OpenmrsAutosuggestProps {
+  name: string;
+  placeholder: string;
+  resource: string;
+  onSuggestionSelected: (field: string, value: string) => void;
+}
+
+export const OpenmrsAutosuggest: React.FC<OpenmrsAutosuggestProps> = ({
+  name,
+  placeholder,
+  resource,
+  onSuggestionSelected,
+}) => {
+  const [suggestions, setSuggestions] = useState([]);
+  const [baseUrl, _] = useState('/ws/rest/v1/');
```

Comment:
You don't need to use `useState` in this kind of scenario. You could just do `const baseUrl = '/ws/rest/v1/';`.

Reply:
Thanks. Will fix.